uber/models.py

```diff
@@ -492,13 +492,10 @@ def get_account_by_email(self, email):
         def no_email(self, subject):
             return not self.query(Email).filter_by(subject=subject).all()
 
-        def lookup_attendee(self, full_name, email, zip_code):
-            words = full_name.split()
-            for i in range(1, len(words)):
-                first, last = ' '.join(words[:i]), ' '.join(words[i:])
-                attendee = self.query(Attendee).iexact(first_name=first, last_name=last, email=email, zip_code=zip_code).all()
-                if attendee:
-                    return attendee[0]
+        def lookup_attendee(self, input_first_name, input_last_name, email, zip_code):
```

Comment:
Whoops! It looks like you renamed the variables in the line below, but not in the function definition. Eli didn't catch this but can you fix it real quick and submit another PR? It won't work right without that fix.